README.md

```diff
@@ -10,3 +10,17 @@ in a time series fashion to provide additional insights about their project.
 # Vizr Installation and Upgrades
 
 See [INSTALL.md](INSTALL.md).
+
+# Vizr Permissions
```

Comment:
Okay, I think I have this right now. I included some documentation here, and updated the `Vizr.php` phpdocs.

Reply:
It's unfortunate that we never really discussed how we wanted this to behave. I'm okay with the behavior described below, but it doesn't match the behavior of the plugin, which doesn't require any special configuration to simply view charts. It would be good if we used tomorrow's review as a opportunity to discuss what we want as a group, although Liz unfortunately won't be there.

Reply:
Good point, this does require an additional step. Now that I understand how this works, that would be an easy change to just have `redcap_module_link_check_display` return `$link` and don't allow module configuration in "Control Center". It would be a good idea to discuss in review and/or when Liz is back.